publish-barshare-gbfs/main.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# This script regularly publishes the BARShare GBFS to the bbnavi open data portal.
+
+set -e
+
+if [ -z "$MOQO_ACCESS_TOKEN" ]; then
+	1>&2 echo 'missing/empty $MOQO_ACCESS_TOKEN'
+	exit 1
+fi
+if [ -z "$MINIO_ACCESS_KEY" ]; then
+	1>&2 echo 'missing/empty $MINIO_ACCESS_KEY'
+	exit 1
+fi
+if [ -z "$MINIO_SECRET_KEY" ]; then
+	1>&2 echo 'missing/empty $MINIO_SECRET_KEY'
+	exit 1
+fi
+
+# default: 5 minutes
+PUBLISH_INTERVAL="${PUBLISH_INTERVAL:-300}"
+
+export MC_HOST_bbnavi="https://$MINIO_ACCESS_KEY:$MINIO_SECRET_KEY@opendata.bbnavi.de"
+
+dir="$(mktemp -d -t barshare-gbfs.XXXXXX)"
+
+set -x
+
+while true; do
+	# We sleep first so that, if the GBFS generation fails contantly, we don't DOS the Moqo API.
+	sleep "$PUBLISH_INTERVAL"
+
+	python moqoToGBFS.py \
+		-c BARshare -b 'https://portal.moqo.de/api_aggregator/' \
```

Comment:
Paths in gbfs.json need to be absolute paths, that's why I introduced a new argument. Note the changed argument names (`b` became `s`):
```suggestion
		-c BARshare -s 'https://portal.moqo.de/api_aggregator/' \
		-b 'https://opendata.bbnavi.de/barshare' \
```

Reply:
Then `README.md` should be adapted as well, right?
https://github.com/bbnavi/moqo2gbfs/blob/83fb0f8c6587a51c5fe6bca92e2c740e6857956c/README.md

Reply:
Indeed. Thanks for flagging this...

Reply:
Is solved via https://github.com/bbnavi/moqo2gbfs/commit/c76c5d88ea9636177ae3240e4fb8c5055e0fa4ab